dss/operations/lambda_params.py

```diff
@@ -0,0 +1,291 @@
+    if env_var in environment:
+        prev_value = environment[env_var]
+    environment[env_var] = value
+    set_deployed_lambda_environment(lambda_name, environment)
+    print(f"Success! Set variable in deployed lambda function {lambda_name}:")
+    print(f"    Name: {env_var}")
+    print(f"    Value: {value}")
+    if prev_value:
+        print(f"    Previous value: {prev_value}")
+
+
+def unset_lambda_var(env_var: str, value, lambda_name: str) -> None:
+    """Unset a single variable in the environment of the specified lambda function"""
+    environment = get_deployed_lambda_environment(lambda_name, quiet=False)
+    try:
+        prev_value = environment[env_var]
+        del environment[env_var]
+        set_deployed_lambda_environment(lambda_name, environment)
+        print(f"Success! Unset variable in deployed lambda function {lambda_name}:")
+        print(f"    Name: {env_var} ")
+        print(f"    Previous value: {prev_value}")
+    except KeyError:
+        print(f"Nothing to unset for variable {env_var} in deployed lambda function {lambda_name}")
+
+
+def print_lambda_env(lambda_name, lambda_env):
+    """Print the environment variables set in a specified lambda function"""
+    print(f"\n{lambda_name}:")
+    for name, val in lambda_env.items():
+        print(f"{name}={val}")
+
+
+lambda_params = dispatch.target("lambda", arguments={}, help=__doc__)
+
+
+json_flag_options = dict(
+    default = False, action="store_true", help="format the output as JSON if this flag is present"
+)
+dryrun_flag_options = dict(
+    default = False, action="store_true", help="do a dry run of the actual operation"
+)
+
+
+@lambda_params.action(
+    "list",
+    arguments={
+        "--json": json_flag_options,
+    }
+)
+def lambda_list(argv: typing.List[str], args: argparse.Namespace):
+    """Print a list of names of each deployed lambda function"""
+    lambda_names = list(get_deployed_lambdas(quiet=args.json))
+    if args.json:
+        print(json.dumps(lambda_names, indent=4, default=str))
+    else:
+        for lambda_name in lambda_names:
+            print(lambda_name)
+
+
+@lambda_params.action(
+    "environment",
+    arguments={
+        "--json": json_flag_options,
+        "--lambda-name": dict(
+            required=False,
+            help="specify the name of a lambda function whose environment will be listed"
+        )
+    }
+)
+def labmda_environment(argv: typing.List[str], args: argparse.Namespace): 
+    """Print out the current environment of deployed lambda functions"""
+    if args.lambda_name:
+        lambda_names = [args.lambda_name]  # single lambda function
+    else:
+        lambda_names = list(get_deployed_lambdas())  # all lambda functions
+
+    # Iterate over lambda functions and get their environments
+    d = {}
+    for lambda_name in lambda_names:
+        lambda_env = get_deployed_lambda_environment(lambda_name, quiet=args.json)
+        d[lambda_name] = lambda_env
```

Comment:
Consider skipping this assignment for non-deployed functions. Otherwise the output looks weird:
```
>scripts/dss-ops.py lambda environment --lambda-name dss-index-de
WARNING:dss.operations.lambda_params:dss-index-de is not a deployed lambda function

dss-index-de:
```

Reply:
Done